pkg/client-go/kubernetes/client.go

```diff
@@ -44,20 +46,18 @@ type kubernetesClient struct {
 
 // NewKubernetesClient creates a KubernetesClient
 func NewKubernetesClient(kubeconfig, apiserver string) (Client, error) {
-	var config *rest.Config
-	var err error
-	if config == nil {
-		config, err = rest.InClusterConfig()
-		if err != nil {
-			if kubeconfig == "" {
-				kubeconfig = filepath.Join(homedir.HomeDir(), ".kube", "config")
-			}
-			config, err = clientcmd.BuildConfigFromFlags(apiserver, kubeconfig)
-			if err != nil {
-				return nil, err
-			}
+	if !fileutil.IsExist(kubeconfig) {
```

Comment:
```suggestion
	if kubeconfig == “” || !fileutil.IsExist(kubeconfig) {
```

Reply:
fileutil.IsExist("")  is false

Reply:
> fileutil.IsExist("") is false

Add check will prevent some heavy sys-calls